### DISCUSSION TITLE
org.strongswan.charon-xpc XCODE

### DISCUSSION THREAD

**Discussion by mister5111** [strongswan/strongswan#747] [Q&A]
When building through xcode, I get this error

`/Users/air/Downloads/strongswan-f4bfdec21f3400d8de3872cad3b8f00c3499471b/src/frontends/osx/<built-in>:1:10: '../../../config.h' file not found

![Снимок экрана 2021-11-11 в 09 45 57](https://user-images.githubusercontent.com/89128897/141258656-062d94fd-6e07-4a1a-8770-828f450c3f36.png)

**Comment by tobiasbrunner** [comment]
See #743. Follow the instructions (i.e. run `configure` etc.) when still trying to build that code.
